Formula/quickjs.rb

```diff
@@ -0,0 +1,25 @@
+class Quickjs < Formula
+  desc "Small and embeddable JavaScript engine"
+  homepage "https://bellard.org/quickjs/"
+  url "https://bellard.org/quickjs/quickjs-2019-07-09.tar.xz"
+  sha256 "350c1cd9dd318ad75e15c9991121c80b85c2ef873716a8900f811554017cd564"
+
+  def install
+    system "make"
+    bin.install "qjscalc"
```

Comment:
Please run `make install` instead of installing files manually

Reply:
As I mentioned this causes problems:
```
mkdir -p "/usr/local/bin"
install -m755 -s qjs qjsc qjsbn qjsbnc "/usr/local/bin"
install: /usr/local/bin/qjs: Operation not permitted
make: *** [install] Error 71

Reply:
`make install prefix=#{prefix}`